app/open.js

```diff
@@ -34,22 +37,37 @@ define(['app/database', 'jquery', 'bootstrap', 'xlsx', 'parsley', 'bootstrap-sel
 			return false; // Don't submit form
 		})
 		.on('form:success', function() {
-			console.log("done");
-			process_wb(JSON.parse(saveData));
+			// Save data into the database
+			var name = $('#uploadForm').find('input[name="name"]').val();
```

Comment:
If you write a comment like this, it is generally a sign that you'd be better of with a function.